apps/readest-app/src/app/reader/components/FoliateViewer.tsx

```diff
@@ -119,6 +122,76 @@ const FoliateViewer: React.FC<{
     );
   };
 
+  const makeImageOverlayScript = (bookKey: string) => {
```

Comment:
There’s already a utility function in the foliate-js library that handles this, and we prefer to reuse existing functionality from foliate-js whenever possible.